sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidPlanner.java

```diff
@@ -380,13 +380,7 @@ public static DruidException translateException(Exception e)
         }
       }
 
-      return DruidException.forPersona(DruidException.Persona.DEVELOPER)
-                           .ofCategory(DruidException.Category.UNCATEGORIZED)
-                           .build(
-                               inner,
-                               "Unable to parse the SQL, unrecognized error from calcite: [%s]",
-                               inner.getMessage()
-                           );
+      return InvalidSqlInput.exception(inner.getMessage());
```

Comment:
this exception creation clips the stacktrace - it only retains the source exceptions message